Services/AccessControl/classes/class.ilObjRole.php

```diff
@@ -508,24 +508,28 @@ function getCountMembers()
 	static function _getTranslation($a_role_title)
 	{
 		global $lng;
-		
-		$test_str = explode('_',$a_role_title);
 
-		if ($test_str[0] == 'il') 
-		{
-			$test2 = (int) $test_str[3];
-			if ($test2 > 0)
-			{
-				unset($test_str[3]);
-			}
+		$test_str = self::_removeObjectId($a_role_title);
 
-			return $lng->txt(implode('_',$test_str));
+		if (preg_match("/^il_./", $test_str))
+		{
+			return $lng->txt($test_str);
 		}
 		
 		return $a_role_title;
 	}
 	
-	
+	public static function _removeObjectId($a_role_title) {
```

Comment:
I need this public because of use in ilObjectRolePermissionTableGUI.php line 685.